README.rst

```diff
@@ -1,28 +1,42 @@
-+----------------------+------------------------------------------------------------------------------------------------------------+
-| master branch status | |BuildStatus| |Docs| |MITLicense| |CoverageStatus|                                                         |
-+----------------------+------------------------------------------------------------------------------------------------------------+
-| GitHub status        | |OpenIssues| |OpenPullRequests|                                                                            |
-+----------------------+------------+-----------------------------------+----------------------------+------------------------------+
-|                      | **Name**   | **Driver Version Tested Against** | **PyPI Version**           | **Supported Python Version** |
-|                      +------------+-----------------------------------+----------------------------+------------------------------+
-| Versions             | NI-DCPower | 17.6.0                            | |nidcpowerLatestVersion|   | |nidcpowerPythonVersion|     |
-|                      +------------+-----------------------------------+----------------------------+------------------------------+
-|                      | NI-DMM     | 17.1.0                            | |nidmmLatestVersion|       | |nidmmPythonVersion|         |
-|                      +------------+-----------------------------------+----------------------------+------------------------------+
-|                      | NI-FGEN    | 17.0.0                            | |nifgenLatestVersion|      | |nifgenPythonVersion|        |
-|                      +------------+-----------------------------------+----------------------------+------------------------------+
-|                      | NI-SCOPE   | 17.0.2                            | |niscopeLatestVersion|     | |niscopePythonVersion|       |
-|                      +------------+-----------------------------------+----------------------------+------------------------------+
-|                      | NI-SWITCH  | 17.0.0                            | |niswitchLatestVersion|    | |niswitchPythonVersion|      |
-|                      +------------+-----------------------------------+----------------------------+------------------------------+
-|                      | NI-ModInst | 17.0.0                            | |nimodinstLatestVersion|   | |nimodinstPythonVersion|     |
-+----------------------+------------+-----------------------------------+----------------------------+------------------------------+
+Overall Status
```

Comment:
Looking at http://nimi-python.readthedocs.io/en/docs-session/

I would do the following:

* Under [Documentation](http://nimi-python.readthedocs.io/en/docs-session/#documentation), have links to the top level per-driver documentation, rather than expanding the whole tree at the end.
* Make [additional documentation](http://nimi-python.readthedocs.io/en/docs-session/#additional-documentation) a smaller heading under Documentation, rather than a peer.

Reply:
Updated per our conversation